spec/acceptance/message_actions_spec.rb

```diff
@@ -193,7 +211,7 @@
     )
     expect(nothing.messages.size).to eq 0
 
-    sleep(5)
+    sleep(7)
```

Comment:
Fair enough, a comment in the code would be nice for the next person that wonders "why 7 seconds?"